ext/jenkins/beaker-tests-source.sh

```diff
@@ -32,6 +32,16 @@ mkdir vendor
 
 bundle install --path vendor/bundle --without test
 
+# Install a copy of leiningen
+if [ -d "leiningen" ];
+then
+  rm -rf leiningen
+fi
+mkdir leiningen
+
+wget https://raw.githubusercontent.com/technomancy/leiningen/stable/bin/lein -O leiningen/lein
```

Comment:
Fair points, the reason why we had this script in the first place was the differences between stable & master and leiningen requirements now - I guess I overlooked the fact the slaves already have a working leiningen today. But I can adjust this, shouldn't be a problem.